src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import 'header';
+@import 'variables';
+@import 'product_card';
+@import 'stars_rating';
+@import 'cards_container';
+
+html {
+  font-family: Roboto, sans-serif, 'Times New Roman';
```

Comment:
It's not recommended to use more than one font family for your project. You should stick with one font family and remove the others.